### PR TITLE
Add force properties to Either

### PR DESCRIPTION
**Pull request by willsb**
Squash whenever

### CODE REVIEW

File: Toggl.Multivac/Either.cs [hunk 1]
```diff
@@ -8,6 +8,10 @@ public sealed class Either<TLeft, TRight>
         private readonly TLeft left;
         private readonly TRight right;
 
+        public TLeft ForceLeft => useLeft ? left : throw new InvalidOperationException();
```
Comment (amulware) [inline Toggl.Multivac/Either.cs]: I'd prefer to skip the `Force`. It just makes it verbose without adding value in my opinion.

File: Toggl.Multivac/Either.cs [hunk 2]
```diff
@@ -8,6 +8,10 @@ public sealed class Either<TLeft, TRight>
         private readonly TLeft left;
         private readonly TRight right;
 
+        public TLeft ForceLeft => useLeft ? left : throw new InvalidOperationException();
+
+        public TRight ForceRight => useLeft ? throw new InvalidOperationException() : right;
```
Comment (amulware) [inline Toggl.Multivac/Either.cs]: Also, we should probably have proper exception messages, no?

File: Toggl.Multivac/Either.cs (hunk 1 above)
Comment (willsb) [inline Toggl.Multivac/Either.cs]: There already is a member called `Left`/`Right`. Should we rename it to what instead?

File: Toggl.Multivac/Either.cs (hunk 2 above)
Comment (willsb) [inline Toggl.Multivac/Either.cs]: Sure thing

File: Toggl.Multivac/Either.cs (hunk 1 above)
Comment (amulware) [inline Toggl.Multivac/Either.cs]: Oh. The statics. They could be renamed to WithRight: Either<string, Error>.WithRight("hi"). That is a common pattern for static initializers anyways. If you really don't like that though, then leave it like it is.

Comment (willsb) [inline Toggl.Multivac/Either.cs]: I prefer the `WithRight` thingy, I'll stick to it

**Pull request merged**